### ISSUE TITLE
WebEngine (Chromium) Images not Working

### ISSUE THREAD

**Issue by ossi-saukko** [retext-project/retext#533]
When using the Qt WebEngine for preview, local images are not loaded.

This can be fixed by commenting or removing the following lines from **webenginepreview.py** file:
``` python
        settings = self.settings()
        settings.setAttribute(QWebEngineSettings.LocalContentCanAccessFileUrls, False)
```
I guess there was a reason to add these lines, so this may cause bug or security issue elsewhere. If you have multiple settings there, only remove or enable the `LocalContentCanAccessFileUrls` setting.

**Comment by mitya57** [comment]
Thank you for the report!

Looks like it worked with Qt 5.14 but stopped working with Qt 5.15. I will investigate further.

**Comment by mitya57** [comment]
I am terribly sorry for the delay. This is now fixed in master branch, maybe you can check if the fix works for you?

**Comment by ossi-saukko** [comment]
Thank you for the fix. I tested this fix from the master branch, and it is working on my system without problems. Good work :-)

**Comment by mitya57** [comment]
Thanks for testing! Pushed the fix also to retext-7.1 branch.
